.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,49 @@
+name: Bug Report
+description: Report Xplorer bug
+labels: [bug]
+body:
+    - type: textarea
+      attributes:
+          label: Description
+          description: A clear and concise description of what the bug is.
+      validations:
+          required: true
+    - type: textarea
+      attributes:
+          label: Steps To Reproduce
+          description: Steps to reproduce the behavior.
+          placeholder: |
+              1. Go to '....'
+              2. Click on '....'
+              3. Scroll down to '....'
+              4. See the error
+      validations:
+          required: false
+    - type: textarea
+      attributes:
+          label: Expected behavior
+          description: A clear and concise description of what you expected to happen.
+      validations:
+          required: true
+    - type: input
+      id: app-version
+      attributes:
+          label: Xplorer Version
+          description: What version of Xplorer are you using?
+          placeholder: v0.1.0
```

I suggest drop-downs so that the version need not be invalid, although it requires bumping on every release.
```suggestion
- type: dropdown
    id: app-version
    attributes:
      label: Xplorer Version
      description: "What version of Xplorer are you running?"
      options:
        - Built from source
        - v0.1.0
```
And if you are wondering, yes, I collaborated on another repo and got the code from there.

---

Good idea, however as described we would need to bump every version on release, which imo isn't really effective though... 😅

---

Oh yeah, I forgot indentation.

---

Sorry, I think it was lag, so the comment came in late. Maybe, by using workflows, bumping can happen, but idk, if you need it, you can add it.